packages/schemas/alterations/next-1665300135-sign-in-sign-up.ts

```diff
@@ -104,7 +104,7 @@ const alteration: AlterationScript = {
 
     /* eslint-disable @silverhand/fp/no-mutating-methods */
     if (data) {
-      const { sign_in_methods: signInMethods } = data;
+      const { signInMethods } = data;
```

Comment:
should this be a method with unit tests? cc @simeng-li @wangsijie

Reply:
No, I don't think it is easy to test

Reply:
Not related to this PR currently.
Discuss offline and create an issue when needed.

https://silverhand-io.slack.com/archives/C02A8G4HVAM/p1666064801635719